portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+import com.google.sps.data.Comment;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+
+import java.io.IOException;
+import java.util.*;
+
```

Comment:
ditto.

btw, meaning of ditto in case you don't know: https://www.google.com/search?q=ditto&oq=ditto&aqs=chrome..69i57j0l4j46j0l2.810j0j7&sourceid=chrome&ie=UTF-8